apps/communications/contacts/js/views/list.js

```diff
@@ -808,6 +813,18 @@ contacts.List = (function() {
 
     loadedContacts[contact.id][group] = contact;
 
+    // If the contact is in the cache but nothing changed, we bail out.
+    // Otherwise, we remove the DOM node and let the flow continue so we
+    // can append the new node with the updated information.
+    if (inCache) {
+      if (Cache.getContact(contact.id) === ph.innerHTML) {
```

Comment:
I'm wondering if we just should check the innerHTML, or also the node data attributes. We put a lot of information there that could vary, and make things depending on data attributes fail. (Just a random though)